datalad/support/archives.py

```diff
@@ -28,7 +28,7 @@
 import random
 
 from .locking import lock_if_check_fails
-from ..utils import any_re_search, assure_bytes
+from ..utils import any_re_search, assure_bytes, rmdir
```

Comment:
Multiple imports per line are a pain wrt code maintenance IMHO

Reply:
I actually started to go the round of "multiple imports per import statement", but I guess not in full...  What do you think about the convention alike
```python
from ..utils import (
    any_re_search,
    assure_bytes,
    rmdir,
)
```
which allows to minimize the diff, and also minimize needed changes while tuning from which module to import ?